intelmq/tests/bots/outputs/templated_smtp/test_output.py

```diff
@@ -87,7 +86,7 @@ def test_malformed_attachment_spec(self):
              unittest.mock.patch('smtplib.SMTP.close'):
             self.run_bot(allowed_error_count=1)
         self.sysconfig["attachments"] = saved_attachments
-        self.assertRegexpMatches(self.loglines_buffer,
+        self.assertRegex(self.loglines_buffer,
```

Comment:
Is that intended?

Reply:
https://docs.python.org/3/library/unittest.html?highlight=assertregexpmatches#unittest.TestCase.assertRegex
> Changed in version 3.2: The method assertRegexpMatches() has been renamed to assertRegex().